packages/reports/src/Components/Form/CompositeCalendar/index.js

```diff
@@ -1,3 +1,3 @@
-import CalendarIcon from './calendar-icon';
+import CompositeCalendar from './composite-calendar.jsx';
 
-export default CalendarIcon;
+export default CompositeCalendar;
```

Comment:
why you change this file?

Reply:
I assume the file should export `CompositeCalendar` since it's in the respective folder, but mistakenly it was written to export the wrong component.

Reply:
That was my bad, I'll change it 🙏🏼